web/src/lib/components/NostrButton.svelte

```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
+    import {onDestroy} from "svelte";
+    import {Pool} from "$lib/nostr/pool";
+    import {getChannelIdForStallOwner, pmChannelNostrRoomId} from '$lib/nostr/utils'
+    import {user} from "$lib/stores";
+
+    export let pmURL: string | null;
+
+    const pool: Pool = new Pool();
+
+    function postToNostr(location) {
```

Comment:
you could do `location : 'stall' | 'mktSquare' | 'nostrFeed'` - so the IDE will let us know if we pass the wrong values.

Reply:
Ah! I didn't thought you could do that. Cool, I'll use it more. Thanks!